### ISSUE TITLE
Empty passphrase error

### ISSUE THREAD

**Issue by EpLiar** [FiloSottile/age#108]
## Environment

* OS: macOS Catalina
* age version:
Age from Homebrew

## What were you trying to do
Encrypt file by a passphrase, but leave an empty, it gives me an error.

## What happened

```
$ age -p go.sum > go.sum.age
Enter passphrase:
Error: empty scrypt password
[ Did age not do what you expected? Could an error be more useful? Tell us: https://filippo.io/age/report ]
$ ./age -p go.sum > go.sum.age  # age by go build
Enter passphrase (leave empty to autogenerate a secure one):
Using the autogenerated passphrase "execute-have-field-bronze-inform-ill-noodle-save-can-leave".
```

**Comment by Fastidious** [comment]
See #78.

**Comment by FiloSottile** [comment]
This is fixed in v1.0.0-beta5
